viewing_party/party.py

```diff
@@ -1,23 +1,219 @@
+import statistics
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+def create_movie(title=None, genre=None, rating=None):
+    """
+    Creates a dictionary with movie details
+    :params: str: title - movie title; default=None
+             str: genre - movie genre; default=None
+             str: rating - movie rating; default=None
+    :returns: new_movie - dictionary
+    """
+    dets = [title, genre, rating]
+    # create dictionary with movie details
+    new_movie = {}
+    keys = ["title", "genre", "rating"]
+    for key, det in zip(keys, dets):
+        new_movie[key] = det
+    # handle missing movie details
+    if any(x == None for x in dets):

```

Comment:
nice concise use of any() and this list checking syntax!

Reply:
Could this check have occurred earlier in the function, in order to be more efficient and skip unused code?

Reply:
Also, good check for None; other cases to validate could include a blank string, ""